apps/annotation/opencv_annotation.cpp

```diff
@@ -218,6 +218,26 @@ vector<Rect> get_annotations(Mat input_image)
 
 int main( int argc, const char** argv )
 {
+    String filename = "animated-webp-supported.webp";
```

Comment:
That is debug code right? Maybe surround it with `#if 1` so that we know it is to delete later.